npm/webpack-dev-server/cypress/e2e/next.cy.ts

```diff
@@ -1,7 +1,7 @@
 /// <reference path="../support/e2e.ts" />
 import type { ProjectFixtureDir } from '@tooling/system-tests/lib/fixtureDirs'
 
-const WEBPACK_REACT: ProjectFixtureDir[] = ['next-11', 'next-12', 'next-11-webpack-4', 'next-12.1.6']
+const WEBPACK_REACT: ProjectFixtureDir[] = ['next-11', 'next-12', 'next-11-webpack-4', 'next-12.1.6', 'next-13']
```

Comment:
While we are here, are we able to drop next-12? Just to keep things nice and fast.We've got next-12.1.6, should cover us. Even if next-12.0.0 isn't working, we could just ask people to update to 12.1.6.

Alternatively, we could drop next-11 (two majors behind now). I am not sure if we actually landed somewhere on EOLing official support for majors, though.

Understand that the webpack resolution is different between next-12 and next-12.1.6, so if you think it's not worth the risk, that's fine.